setup.py

```diff
@@ -1,7 +1,10 @@
+import setuptools
 from os import path
+from distutils.command.build_ext import build_ext as DistUtilsBuildExt
 from setuptools import setup, find_packages
 
-VERSION = '0.2.6'
+VERSION = '0.3.6'
```

Comment:
If we're bumping this, it should 0.2.7 right?

Reply:
Ah got it, will fix.